### ISSUE TITLE
populateDataToJS - support for storeAs parameter

### ISSUE THREAD

**Issue by darrenfurr** [prescottprue/react-redux-firebase#216]
**Do you want to request a *feature* or report a *bug*?**  
Feature

**What is the current behavior?**
storeAs currently only works for dataToJS to handle multiple queries, however, populateDataToJS does not.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net or similar.**
This currently does not work. I have to remove the populates & just use dataToJS

```
const populates = [
  { child: 'project_versions', root: 'project_versions', keyProp: 'versionId' }
]

// Get apps, versions, org by orgKey
@firebaseConnect(props => [
  { path: 'projects', storeAs: 'org_projects', queryParams: ['orderByChild=organization', `equalTo=${props.orgKey}`], populates}
])
@connect(({ firebase }, props) => ({
  apps: populatedDataToJS(firebase, '/org_projects', populates),
}))
```

**What is the expected behavior?**
populateDataToJS supports storeAs

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**
v1.4.4

**Comment by prescottprue** [comment]
Looking into this.

`populate` in v2 currently works with `storeAs` if you need an intermediate solution.

**Comment by prescottprue** [comment]
Great catch! Released in [`v1.4.5`](https://github.com/prescottprue/react-redux-firebase/releases/tag/v1.4.5).

 More extensive testing on this to come, so we don't miss anything else with `storeAs`.
